Training/PythonWebServer/index.html

```diff
@@ -9,6 +9,89 @@
     <script src="/resources/index.js" type="text/javascript"></script>
 </head>
 <body>
-    <h1>Hello world!</h1>
+	<div class='wrapper'>
+        <button class="button" onclick="loadDoc()">UPDATE</button>
+
+    	<div class="display">
+        Time:
+            <p id = "Time1"></p> 
+            <p id = "Time2"></p> 
+            <p id = "Time3"></p> 
+            <p id = "Time4"></p>
+            <p id = "Time5"></p> 
+            <p id = "Time6"></p> 
+            <p id = "Time7"></p> 
+            <p id = "Time8"></p>
+        <br><br><br>
+    	</div>
+
+    	<div class="display">
+        Total CPUs: 
+        <p id = "totalCPU"></p>
+        <br> <br>
+    	CPU Usage:
+            <p id = "cpuUsage1"></p> 
+            <p id = "cpuUsage2"></p> 
+            <p id = "cpuUsage3"></p> 
+            <p id = "cpuUsage4"></p>
+    	</div>
+
+    	<div class="display">
+    	Total RAM:
+            <p id = "totalRAM1"></p>
+            <p id = "totalRAM2"></p>
+            <p id = "totalRAM3"></p>
+            <p id = "totalRAM4"></p>
+            <p id = "totalRAM5"></p>
+    		<br><br>
+    	Used RAM:
+            <p id = "usedRAM1"></p>
+            <p id = "usedRAM2"></p>
+            <p id = "usedRAM3"></p>
+            <p id = "usedRAM4"></p>
+            <p id = "usedRAM5"></p>
+    	</div>
+    </div>
+    	<br><br>
+
+        <script>
+        function loadDoc() {
```

Comment:
In a `<script>` tag, usually javascript is written in between the tags to run on the page, which is what you've done. But the problem with script tags is that it mixes some different languages in a single file and in general is less organized.

Generally, all javascript goes in the javascript files, in this case `index.js`. Putting it in the js files runs it the same as if it was in a `<script>` tag in the html!

And `.js` files aren't magically loaded! For us, the javascript file was loaded in on line 9 in this HTML file.
```html
    <script src="/resources/index.js" type="text/javascript"></script>
```